src/build-deps.txt

```diff
@@ -1,2 +1,3 @@
+time
```

Comment:
If you wouldn't mind the time, can you ad a comment above this like e.g.:
`# A common utility, will be used by e.g. coreos-assembler builds`
?

Reply:
Maybe even `# A common utility, can/will be used for diagnostics in cosa builds`.